UI/data/locale/en-US.ini

```diff
@@ -570,11 +570,16 @@ Basic.Settings.General.SystemTrayHideMinimize="Always minimize to system tray in
 Basic.Settings.General.SaveProjectors="Save projectors on exit"
 Basic.Settings.General.SwitchOnDoubleClick="Transition to scene when double-clicked"
 Basic.Settings.General.StudioPortraitLayout="Enable portrait/vertical layout"
+Basic.Settings.General.Multiview="Multiview"
+Basic.Settings.General.Multiview.MouseSwitch="Change scenes with clicks"
+Basic.Settings.General.Multiview.DrawSourceNames="Draw sources name"
+Basic.Settings.General.Multiview.DrawSafeAreas="Draw safe areas (EBU R 95)"
 Basic.Settings.General.MultiviewLayout="Multiview Layout"
-Basic.Settings.General.MultiviewLayout.Horizontal.Top="Horizontal, Top"
-Basic.Settings.General.MultiviewLayout.Horizontal.Bottom="Horizontal, Bottom"
-Basic.Settings.General.MultiviewLayout.Vertical.Left="Vertical, Left"
-Basic.Settings.General.MultiviewLayout.Vertical.Right="Vertical, Right"
+Basic.Settings.General.MultiviewLayout.Horizontal.Top="Horizontal, Top (2X4X4)"
+Basic.Settings.General.MultiviewLayout.Horizontal.Bottom="Horizontal, Bottom (4X4X2)"
+Basic.Settings.General.MultiviewLayout.Vertical.Left="Vertical, Left (2X4X4)"
+Basic.Settings.General.MultiviewLayout.Vertical.Right="Vertical, Right (4X4X2)"
+Basic.Settings.General.MultiviewLayout.Horizontal.Extended.Top="Horizontal, Top (1_2_1X6X4)"
```

Comment:
The layout measurements here would be confusing to most users, I'm not exactly sure what it means just by reading it. Not sure how it could be improved.

Reply:
Would change the "(aXbXc)" to juse "8 scenes"/"24 scenes" work better?

Reply:
I think it would, as most people won't care about the size/shape of the grid but how many scenes they get out of it. That's a good idea.